tests/management-kf-1.1/README.md

```diff
@@ -0,0 +1,14 @@
+The makefile provides a quick way to install management cluster in Kubeflow
+1.1.0, so that we can verify upgrade experience.
+
+Usage:
+```bash
+# deploy management cluster in Kubeflow 1.1.0
+make test
```

Comment:
nit: While I understand this folder is for testing, `make test` seems a bit odd for the purpose of `deploy management cluster in Kubeflow 1.1.0`

Reply:
It's also creating a gcs bucket resource though. Do you have any suggestions?

Reply:
I don't really have a good suggestion, just feel like the words `test` and `deployment` are not close.
Maybe something like `make test-setup` or `make create-test-cluster`?

Reply:
PTAL, I've refactored tests to leverage some ways to dynamically set variables for makefile